search/search_repository.go

```diff
@@ -616,10 +616,9 @@ func (r *GormSearchRepository) listItemsFromDB(ctx context.Context, criteria cri
 	if parentExists != nil && !*parentExists {
 		where += ` AND
 			id not in (
```

Comment:
use "not exists" instead of "not in"

```
AND NOT EXISTS (
	SELECT wil.target_id FROM work_item_links wil, work_item_link_types wilt
	WHERE wil.link_type_id = wilt.id and wilt.forward_name = 'parent of' 
	AND wil.deleted_at is null
	AND wil.target_id = work_items.id
)
```

Reply:
work_item_links could use a index on (target_id, deleted_at) as well

Reply:
@kwk we assume work_item_link_types will be relatively small? Is that per space when space templates comes in?

Reply:
@aslakknutsen that's better indeed, thanks ! I'll fix that the query and include an index

Reply:
> @kwk we assume work_item_link_types will be relatively small? Is that per space when space templates comes in?

@aslakknutsen link types will be per space template, but yes the amount of them is probably very small.